jar-infer/scripts/android-jar.conf

```diff
@@ -1,8 +1,19 @@
 [android-paths]
 aosp-out-dir = /Volumes/AOSP8.1/out/target/common/obj/JAVA_LIBRARIES/framework_intermediates
-android-jar-path = ~/android-sdk/platforms/android-27/android.jar
+android-stubs-jar = ~/android-sdk/platforms/android-27/android.jar
```

Comment:
Why are these paths relative to the home directory?  Seems brittle

Reply:
This is the configuration for running the `android-jar.py` script manually and will need to be changed locally before running. Perhaps we should rename the committed version of this file as `android-jar.conf.template` and gitignore the `android-jar.conf` files?